tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleProject.java

```diff
@@ -187,6 +187,11 @@ public IPath getOutputBundlePath() {
 
 				CustomModel model = GradleCore.getToolingModel(CustomModel.class, gradleProject);
 
+                if( model == null )
```

Comment:
@haoyisun please make sf for this project before you submit

Reply:
OK